LoopMusic/Source/ViewControllers/LoopFinder/BaseLoopFinderContainerViewController.swift

```diff
@@ -12,6 +12,14 @@ class BaseLoopFinderContainerViewController<Item>: UIViewController {
     var currentItem: Item!
     /// Index of the currently selected item.
     var currentItemIndex: Int = 0
+    /// Original item.
+    var originalItem: Item!
+    /// Flag for viewing manually entered values.
+    var manualMode: Bool = false
+    /// The minimum allowable item index.
+    var minItemIndex: Int {
```

Comment:
Allowing the index to be -1 is a bit unsafe. Why not append the original item to the start of the array?

Reply:
Using -1 lets the original loop be treated specially (e.g. by `updateAutomatic()`), while still naturally imposing an order relative to the other entries. Any suggestions on how to accomplish this by just appending it to the array?

Reply:
You can have a boolean flag if the 0th element is an original item. Then you can still have special logic for it without risking issues with a negative array index.

Reply:
Resolved in e15ec04